packages/mesos/README.md

```diff
@@ -4,6 +4,4 @@ These commits can be found in the repository at <a href="https://github.com/meso
 <li>[09975e642168a47c466e211f02a590b6e3e1e40c] Changed agent_host to expect a relative path.
 <li>[d2cf9ac1dfa0d7d2b5ebcf8f9da86190706bb5cd] Mesos UI: Change paths, pailer, and logs to work with a reverse proxy.
 <li>[3d042bae0431da237d2c109525ea81fe8eb9943c] Revert "Fixed the broken metrics information of master in WebUI."
-<li>[7fdf195b5f3bb4d4a0bd155d1663a96f9eaf4da1] Fixed fetcher to not pick up environment variables it should not see.
```

Comment:
Was this one actually cherry-picked before, but not listed here? Or is this a new one? If it's a new one, why is it before the GPU one and not after?